packages/libs/src/sdk/services/Auth/types.ts

```diff
@@ -24,4 +26,38 @@ export type AuthService = {
    * Get the sender address, used for EntityManager writes
    */
   getAddress: () => Promise<string>
+
+  signUp?: ({
+    email,
+    password
+  }: {
+    email: string
+    password: string
+  }) => Promise<Wallet>
+
+  signIn?: ({
+    email,
+    password,
+    otp
+  }: {
+    email: string
+    password: string
+    otp?: string
+  }) => Promise<Wallet>
+
+  signOut?: () => void
+
+  isSignedIn?: () => void
+}
+
+export type AuthConfig = {
+  /**
+   * The identity service that backs hedgehog authentication
+   */
+  identityService: string
```

Comment:
Same with these, I think these are unique to hedgehog auth and shouldn't be required for the others